src/components/Checkbox/Checkbox.tsx

```diff
@@ -61,10 +67,12 @@ export const Checkbox = React.forwardRef<HTMLInputElement, Props>(
                             width="12px"
                             height="10px"
                             viewBox={CHECKBOX_VIEWBOX}
+                            role="img"
+                            aria-label={indeterminate ? lineLabel : polylineLabel}
                         >
-                            {indeterminate && <line x1="2" y1="6" x2="8" y2="6" />}
-                            {!indeterminate && (checked === undefined || checked === true) && (
-                                <polyline points="1.5 6 3.5 9 8 3" />
+                            {indeterminate && <line x1="2" y1="6" x2="8" y2="6" role={lineLabel} />}
+                            {!indeterminate && (checked === undefined || checked) && (
```

Comment:
I'm not sure about this approach. Maybe better rename labels to roles ?

Reply:
sounds good, less confusing. Especially having that you used `ariaRole` in `ColorValue.tsx`.

Let's rename